activiti-api/activiti-process-model-api-impl/src/main/java/org/activiti/runtime/api/model/impl/ProcessDefinitionMetaImpl.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.runtime.api.model.impl;
+
+import java.util.List;
+
+import org.activiti.runtime.api.model.ProcessDefinitionMeta;
+
+public class ProcessDefinitionMetaImpl implements ProcessDefinitionMeta {
+
+    private String processDefinitionKey;
+    private List<String> usersIds;
+    private List<String> groupIds;
+    private List<String> connectorsIds;
```

Comment:
@erdemedeiros yeah.. that was the idea.. how can we name those? "connectorsNames" maybe ? ServiceTaskImplementations sounds bad.. This is related to the Connector interface

Reply:
I agree with you `ServiceTaskImplementations` sounds bad. `connectorNames` might be an option. I don't like `id` because my first though is UUIDs. BTW what about users and groups? usernames and groupNames?